packages/common/index.ts

```diff
@@ -203,11 +204,13 @@ export type Question = {
   closedAt?: Date;
   questionType?: QuestionType;
   status: QuestionStatus;
+  position: number;
```

Comment:
I'm not sure if position is something we want to be storing in a Question, especially since we'd have to constantly update it whenever the queue changes. Maybe this is something we could derive from the index of a question in a time sorted array?